bot/extensions/moderation/moderation.py

```diff
@@ -0,0 +1,52 @@
+from discord import Member
+import datetime
+from json import loads
+from turtle import title
+from discord.ext.commands.cooldowns import BucketType
+from discord.ext.commands import Cog, command, cooldown, has_permissions
+from discord import Embed
+
+
+class ModerationCog(Cog, name="moderation", description="Collection of administrative use commands."):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='kick', help="Allows a staff member to kick a user based on their behaviour.")
+    @cooldown(1, 5, BucketType.user)
+    @has_permissions(kick_members=True)
+    async def kick(self, ctx, member: Member, reason="No reason"):
+            guild = ctx.guild
+            embed = Embed(title="Grace Moderation - KICK", description=f"{member.mention} has been kicked.", timestamp=datetime.datetime.utcnow())
+            embed.add_field(name="Reason: ", value=str(reason), inline=False)
+            await ctx.reply(embed=embed)
+            await guild.kick(user=member)
+
+    @command(name='ban', help="Allows a staff member to ban a user based on their behaviour.")
+    @cooldown(1, 5, BucketType.user)
```

Comment:
Remove cooldown on ban and kick, add mass ban/kick commands in the future.

Reply:
Solved on the commit below this.